victor_hardware_interface/src/victor_hardware_interface/victor_utils.py

```diff
@@ -127,11 +123,67 @@ def get_joint_impedance_params(stiffness, vel=0.1, accel=0.1):
 
     else:
         rospy.logerr("Unknown stiffness for Joint Impedance")
-        assert(False)
+        assert False
 
     return new_control_mode
 
 
+def get_cartesian_impedance_params(velocity=2.5):
```

Comment:
Where did these numbers come from? Have you tested these?

Reply:
I got them from johnson, pretty sure he's tested them @LemonPi emon

Reply:
Yup these are tested values, but you mentioned that velocity=2.5 as default may be too low? The motions I've been doing are pretty small so it's been appropriate.

Reply:
we can leave it at this, slow by default is better than fast